src/templates/games/game_detail.html

```diff
@@ -34,25 +34,17 @@ <h3 class="text-body-secondary">{{ game.description|linebreaksbr }}</h3>
     </div>
     <div class="col-9">
       <div data-bs-target="#description-list" tabindex="0">
+        <h3 id="description">Description</h3>
```

Comment:
The changes in this file reflect the goal of the issue successfully. You changed the content of the descriptions from a h3 to a paragraph. You've also moved the description to the information section. Everything seems okay here. I just have a small suggestion on the style. Since we've used h1 for the game title, would it be better to use h2 for the attributes of the game instead of h3? For instance, having the title "Description as h2

Reply:
I think this suggestion makes a lot of sense, but when I initially made this change I decided (very subjectively) that `h3` looked better. I think I'm going to keep it for now but would have no problem with someone opening up a new issue to adjust game detail styling in the future.

Reply:
I understand. Thank you for letting me know!